cves/CVE-2013-0870.yml

```diff
@@ -64,7 +64,14 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
+  The is a piece of code that checks the type of thread that is active. It is
+  currently in the wrong place. It is inside of a type check, which it has
+  no business being in. This incorrect nesting brought the system to an 
+  inconsistent state. To summarize, this conditional that catches and prevents
+  midstream thread reconfiguration error, was being limited to one group, when
+  it should be run for all groups. This reconfiguration is important to catch
+  because it is unsupported and causes the inconsistent system state.
```

Comment:
These entire paragraph sounds way too choppy. I think you can add some comma and clauses in there to make it sound better.